src/main/java/sirius/biz/translations/MultiLanguageString.java

```diff
@@ -67,8 +67,8 @@ public MultiLanguageString withFallback() {
      * @param validLanguages the list of language codes to validate against
      * @return the object itself for fluent method calls
      */
-    public MultiLanguageString withValidLanguages(@Nonnull List<String> validLanguages) {
-        this.validLanguages = Collections.unmodifiableList(validLanguages);
+    public MultiLanguageString withValidLanguages(@Nonnull Collection<String> validLanguages) {
+        this.validLanguages = new ArrayList<>(validLanguages);
```

Comment:
is it wanted that it could be altered internally?

Reply:
a copy is created on write and shielded on read